src/app/common/services/form/form.service.ts

```diff
@@ -30,8 +30,8 @@ export const realEstateForm = new FormGroup({
   price: new FormControl('', [Validators.required])
 });
 
-  export const agentForm = new FormGroup({
-    name: new FormControl('', [Validators.required]),
-    email: new FormControl('', [Validators.required, Validators.email]),
-    phoneNumber: new FormControl('', [Validators.required]),
-  });
+export const agentForm = new FormGroup({
+  name: new FormControl('', [Validators.required, Validators.minLength(5), Validators.maxLength(100)]),
+  email: new FormControl('', [Validators.required, Validators.email, Validators.minLength(9), Validators.maxLength(13)]),
```

Comment:
Maybe this should be edited since it's not unsual for an email address to be more than 13 characters long

Reply:
Maybe, but since the api was 13 max length i just followed the rule